src/components/landing-page/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { LANDING_PAGE_FEATURES } from "@/lib/constants/misc";
+import { Button } from "../ui/button";
+import Image from "next/image";
+import { grayBlurDataValue } from "@/lib/utils";
+
+const LandingPage = () => {
+  return (
+    <>
+      <div className="bg-[url('/images/app_landing.jpg')] bg-center md:h-[400px] bg-cover rounded-[12px] p-[40px]">
+        <div className="h-full flex flex-col justify-end">
+          <div className="space-y-[20px]">
+            <h1 className="text-[36px] text-white font-[700] md:text-[48px]">
+              SpendWise helps you spend smarter and save more.
+            </h1>
+            <p className="text-white text-[18px] font-[200]">
```

Comment:
here using custom fonts will solve to use font-weight, then you have to use with names like font-medium, font-bold etc